tests/base.py

```diff
@@ -261,14 +259,22 @@ def minimum_record_count_by_stream(self):
 
         return record_counts
 
-    def dt_to_ts(self, dtime):
+    def strptime_to_timestamp(self, dtime):
         for date_format in self.DATETIME_FMT:
             try:
                 date_stripped = int(time.mktime(dt.strptime(dtime, date_format).timetuple()))
                 return date_stripped
             except ValueError:
                 continue
 
+    def strftime_to_datetime(self, date_str):
+        for date_format in self.DATETIME_FMT:
+            try:
+                date_time = dt.strptime(date_str, date_format)
+                return date_time
+            except ValueError:
+                continue
```

Comment:
I think, below code should perform similarly but be a bit cleaner and more direct!

```
def strftime_to_datetime(self, date_str):
    for date_format in self.DATETIME_FMT:
        try:
            return dt.strptime(date_str, date_format)
        except ValueError:
            pass
    return None
```